src/components/Article.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react"
+import { useParams } from "react-router-dom"
+import { fetchArticle } from "../utils/api-utils"
+
+function Article() {
+    const [article, setArticle] = useState({})
+    const {article_id} = useParams()
+    console.log(article_id)
+    console.log(article)
+
+    useEffect(() => {
+        fetchArticle(article_id).then(({article}) => {
+            setArticle(article)
+        })
+        
+    }, [])
+
```

Comment:
Please add a loading state to this component and use conditional rendering to let the user know that the article is loading.